src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,313 @@
+
+    public class NetworkStatistics : IDisposable
+    {
+        private IntPtr _infoHandle = IntPtr.Zero;
+        private bool _disposed;
+
+        internal NetworkStatistics(IntPtr handle)
+        {
+            Log.Debug(Globals.LogTag, "New Info. Handle: " + handle);
+            _infoHandle = handle;
+        }
+
+        /// <summary>
+        /// Destroy the NetworkStatistics object
+        /// </summary>
+        ~NetworkStatistics()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// A method to destroy the managed NetworkStatistics objects.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        protected void Dispose(bool disposing)
+        {
+            if (_disposed)
+            {
+                return;
+            }
+
+            if(disposing)
+            {
+                // destroy managed resources
+            }
+
+            int ret = Interop.Stc.Info.StatsDestroy(_infoHandle);
+            if (ret == (int)StcError.None)
+            {
+                _infoHandle = IntPtr.Zero;
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Failed to destroy network statistics handle, Error - " + (StcError)ret);
+            }
+
+            _disposed = true;
+        }
+
+        /// <summary>
+        /// A property to get the application ID from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Application ID.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string AppId
+        {
+            get
+            {
+                string appId;
+                int ret = Interop.Stc.Info.GetAppId(_infoHandle, out appId);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get AppId from Info, Error - " + (StcError)ret);
+                    return string.Empty;
+                }
+                return appId;
+            }
+        }
+
+        /// <summary>
+        /// A property to get interface name from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Interface name.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
```

Comment:
Wrong comment. this property only provides getter. So when try set any value to this property. It makes a complication error.